components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,128 @@
+
+:root {
+  --primary-color: #ff7474;
+  --primary-text-color: #ff5454;
+  --background-color: #fde3e3;
+  --border-color: #d3d3d3;
+}
+
+.font {
+  font-family: "Roboto", sans-serif;
+  font-size: 1rem;
+}
+
+.snackbar_main_container {
+  margin: 1rem 0.5rem;
+}
+
+.btn_container{
+  border: 1px solid var(--border-color);
+  padding: 4rem 2rem;
+  border-radius: 1rem;
+
+}
+
+.snackbar_container {
+  background-color: var(--background-color);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 30rem;
+  border-bottom: 4px solid var(--primary-color);
+  position: fixed;
+  bottom: 0.5rem;
+  padding: 1rem 1.5rem;
+  margin: 1rem;
+  border-radius: 1rem;
+  visibility: hidden;
+}
+
+.snackbar_container a {
+  color: var(--primary-text-color);
+  text-decoration: none;
+  font-size: 1.2rem;
+  padding: 0.5rem 0.7rem;
+}
+
+.snackbar_container a:hover {
+  background-color: var(--primary-text-color);
+  color: white;
+}
+
+.baseline_snackbar,
+.stacked_snackbar {
+  left: 50%;
+  transform: translate(-50%, 0px);
```

Comment:
translateX can be used instead of translate

Reply:
Okay.Great suggestion